### DISCUSSION TITLE
2 Row Dock.

### DISCUSSION THREAD

**Discussion by silvanathecat** [LawnchairLauncher/lawnchair#4109] [Ideas]
No clue if this was a feature in Lawnchair V2 as I didn't use it, but the main thing preventing me from using Nova or Lawnchair is only having a single row dock. My workflow is heavily reliant on 14 apps being available at a glance on any homescreen page.

**Comment by greeter9473** [comment]
> No clue if this was a feature in Lawnchair V2 as I didn't use it, but the main thing preventing me from using Nova or Lawnchair is only having a single row dock. My workflow is heavily reliant on 14 apps being available at a glance on any homescreen page.

It is a feature in Lawnchair V2. V2 /Legacy still works just fine. Two row dock along with cover mode folders (icon is first app in folder, tap to open first app / swipe to open folder) are the reasons I've not switched to anything else so far.

**Comment by syborgxx** [comment]
I like how Nova does it with a scrollable dock.

**Comment by SuperDragonXD** [comment]
Hello. I know it's a bit late, but I'll close this to centralize discussion at #3216.
